internal/engine/experiment/riseupvpn/riseupvpn.go

```diff
@@ -68,6 +68,9 @@ type TestKeys struct {
 	TransportStatus map[string]string   `json:"transport_status"`
 }
 
+// TODO(bassosimone,cyberta): make sure the spec mentions
+// the transport_status result key.
```

Comment:
This is the relevant PR: https://github.com/ooni/spec/pull/211

Reply:
Currently under review!

Reply:
I've adapted the spec

Reply:
Thanks, merged!